db/schema.yml

```diff
@@ -805,7 +787,7 @@ container_port_configs:
 - port
 - host_port
 - protocol
-- container_definition_id
+- container_id
```

Comment:
@carbonin Will this cause a problem since it's in the middle of the pack?  I think it's ok, just want to verify.